packages/arb-node-core/cmd/arb-validator/arb-validator.go

```diff
@@ -95,6 +96,17 @@ func startup() error {
 		return nil
 	}
 
+	if config.Core.Test.JustMetadata {
+		storage, err := cmachine.NewArbStorage(config.GetValidatorDatabasePath(), &config.Core)
+		if err != nil {
+			return err
+		}
+		storage.PrintDatabaseMetadata()
+		storage.CloseArbStorage()
+
+		return nil
+	}
+
```

Comment:
Maybe add this to the `cmdhelp` to dedup?

Reply:
:+1: